src/app/public/shared/status/status.component.css

```diff
@@ -0,0 +1,29 @@
+.statusCard {
+  border: 1px solid black;
+  border-radius: 10px;
+  width: 100px;
+  padding-top: 2px;
+  padding-bottom: 2px;
+  font-size: 12px;
+  vertical-align: middle;
+  margin: 10px;
+}
+
+p {
+  padding: 0;
+  margin: 0 !important;
+}
+.forks {
```

Comment:
this can be `statusCard--forks` & `statusCard--watchers` ? This is an example of modifier class :)